03Lesson/01Project/index.html

```diff
@@ -1,9 +1,97 @@
 <!DOCTYPE html>
 <html>
+<meta charset="utf-8">

```

Comment:
No elements are allowed outside of `head` and `body`. Make sure to check W3C validation errors in Atom.